datalake/src/main/java/com/sequenceiq/datalake/converter/SdxDatabaseDrStatusTypeEnumConverter.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.datalake.converter;
+
+import com.sequenceiq.cloudbreak.converter.DefaultEnumConverter;
+import com.sequenceiq.datalake.entity.operation.SdxOperationStatusTypeEnum;
+
+public class SdxDatabaseDrStatusTypeEnumConverter extends DefaultEnumConverter<SdxOperationStatusTypeEnum> {
```

Comment:
I think this class should be renamed also

Reply:
ok